src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -166,7 +167,7 @@ def _add_optional_columns(df: pd.DataFrame) -> pd.DataFrame:
     in_df_cols = set(df.columns)
     if not optional_col_set.issubset(in_df_cols):
         additional_col = list(optional_col_set.difference(in_df_cols))
-        additional_df = pd.DataFrame(columns=additional_col, index=df.index, data=pd.NA)
+        additional_df = pd.DataFrame(columns=additional_col, index=df.index, data=None)
```

Comment:
What is the advantage here?

Reply:
There is no advantage, I did this to please mypy